spec/factories/factories.rb

```diff
@@ -47,5 +46,18 @@
     "person#{n}@example.com"
   end
 
+  factory :comment do

```

Comment:
add Comments generation to seeds.rb, so you will have some populated comments from the beginnning